### PR TITLE
update openshift instructions

### PR DESCRIPTION
**Pull request by pb82**
Update the Openshift Instructions to make use of the Service Catalog & APB. The old instructions were still based on Openshift v2.

### CODE REVIEW

**Comment by matzew** [comment]
:eyes:

File: docs/unifiedpush/ups_userguide/openshift.asciidoc [hunk 1]
```diff
@@ -8,39 +8,46 @@
 [[openshift]]
 == Running the UnifiedPush Server on OpenShift
 
-AeroGear provides an UnifiedPush Server cartridge to make it easy to get it running on OpenShift. If you don't already have an account or don't know how to create applications go to https://www.openshift.com/ first. You can create the UnifiedPush Server instance either with the web tool or the command line tool, both approaches are described below.
+AeroGear provides a link:https://github.com/aerogearcatalog/unifiedpush-apb[Ansible playbook bundle] to make it easy to get running on OpenShift.
```
Comment (matzew) [inline docs/unifiedpush/ups_userguide/openshift.asciidoc]: Ansible Playbook Bundle (APB) (capital letters)

File: docs/unifiedpush/ups_userguide/openshift.asciidoc [hunk 2]
```diff
@@ -8,39 +8,46 @@
...
+If you don't already have an account you can sign up link:https://www.openshift.com/pricing/index.html[here]. You can also use one of the various ways to run OpenShift on your own hardware like link:https://github.com/minishift/minishift[Minishift], link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up] or the link:https://github.com/aerogear/mobile-core[Mobile Application Platform].
 
 
 WARNING: It's important that immediately after creating an UnifiedPush Server instance you open the AdminUI and login to reset the password. If this is not done anyone can easily gain admin rights to your UnifiedPush Server instance.
```
Comment (matzew) [inline docs/unifiedpush/ups_userguide/openshift.asciidoc]: this is not needed, on our Mobile Platform, sicne all is just OAuth proxy

File: docs/unifiedpush/ups_userguide/openshift.asciidoc [hunk 3]
```diff
@@ -8,39 +8,46 @@
...
 
-image:http://launch-shifter.rhcloud.com/launch/light/LAUNCH ON.svg["Run me on Openshift",link="https://openshift.redhat.com/app/console/application_type/quickstart!31"]
-
-Now, just fill in the form and click on +Create Application+! This creates an UnifiedPush Server instance with a configured _MySQL 5.5_ database. Click on +Continue to the application overview page+.
-Under the list of applications you should find your new UnifiedPush Server instance and the status should be _Started_. Click on it to open the UnifiedPush Server's homepage.
-
-=== Create an instance using OpenShift's command line interface
+[source,bash]
+----
+$ oc adm policy add-cluster-role-to-user access-asb-role developer
```
Comment (matzew) [inline docs/unifiedpush/ups_userguide/openshift.asciidoc]: `cluster-admin` is needed too, no ?

File: docs/unifiedpush/ups_userguide/openshift.asciidoc [hunk 4]
```diff
@@ -8,39 +8,46 @@
...
+----
+$ apb bootstrap
+----
 
-If you are interested to learn more about +rhc+, checkout the following resources:
+=== Create an instance using the Ansible Service Broker
 
-* link:https://www.openshift.com/blogs/using-rhc-to-manage-paas-apps[How to Manage Apps in the Cloud with RHC]
-* link:https://access.redhat.com/documentation/en-US/OpenShift/2.0/html-single/User_Guide/index.html[OpenShift Online - User Guide]
+Open the Service Catalog and you should see an entry for `AeroGear UnifiedPush Server`. Select it, pick a namespace to deploy it and a Wizard will guide you through the setup process.
```
Comment (matzew) [inline docs/unifiedpush/ups_userguide/openshift.asciidoc]: perhaps we do a screenshot here, of the latest UI ?

**Comment by matzew** [comment]
minor comments, but thanks for taking this on, @pb82

**Comment by pb82** [comment]
New PR for the apb repo: https://github.com/aerogearcatalog/unifiedpush-apb/pull/40

**Pull request closed without merging**